WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,55 @@
+
+## Daily routine
+
+1. [Check code review requests](https://github.com/zkSNACKs/WalletWasabi/pulls). __Reviews / Awaiting review from you__.
+2. Check [priority issues and PR-s!](https://github.com/zkSNACKs/WalletWasabi/labels/priority). Take it or set Assignees and Labels if you can.
+3. Check your [own PR-s and answer](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+repo%3AzkSNACKs%2FWalletWasabi+) if necessary. __Click Created__.
+   - If you need help use __@mention__.
+   - If you need review __set Reviewers__.
+4. Check [assigned issues](https://github.com/zkSNACKs/WalletWasabi/issues) __Click Assignee / Select your profile__.
+5. Check [mentions](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+repo%3AzkSNACKs%2FWalletWasabi+) __Click Mentioned__.
+6. Check the bell.
+7. Work on anything you want - assign it to yourself.
+
+## Weekly routine
+
+- Wednesday: Developer meeting. Use [dev meeting board](https://github.com/orgs/zkSNACKs/projects/1) to manage tasks.
+- Pair programming hour.
+
+## Monthly routine
+
+- Go through all Issues and Pull requests and triage them.
```

Comment:
They wouldn't do this. I think there's no way anyone would do it other than let's say me, you and yahiheb, so I think it shouldn't be in this document as we can see already that it won't work happen :/

Reply:
I will remove them in favor of now stressing our devs with this.